docusaurus-docs/conda-store-ui/tutorials/setup.md

```diff
@@ -47,11 +47,9 @@ conda-store-server --standalone
 
```

Comment:
Please move changes in this file into a separate PR. To me, these look unrelated to the environment variables change, which means they can be merged separately (and likely much faster as well).

Reply:
Thanks for your review :)

I did mention in the PR description that it includes some minor, miscellaneous clean-up, so I thought this would be alright. 

However, I do agree we should aim for cleaner+specific PRs. I'll revert these updates.